nextpnr-xilinx/build.sh

```diff
@@ -15,13 +15,15 @@ make install
 DEVICES="xc7a35tcsg324-1 \
          xc7a35tcpg236-1 \
          xc7z010clg400-1 \
-         xc7z020clg484-1"
+         xc7z020clg484-1 \
+         xc7a100tcsg324-1 \
+         xc7a200tsbg484-1"
 
 SHARE_DIR=${PREFIX}/share/nextpnr-xilinx
 mkdir -p $SHARE_DIR
 
 # Compute data files for nextpnr-xilinx
 for device in $DEVICES; do
-    python xilinx/python/bbaexport.py --device $device --bba $SHARE_DIR/$device.bba
+    pypy3 xilinx/python/bbaexport.py --device $device --bba $SHARE_DIR/$device.bba
```

Comment:
This should be dependent on some environment variable.

Also I wonder if this highlights the bba should be in a separate package to the binaries?

Reply:
> Also I wonder if this highlights the bba should be in a separate package to the binaries?

You mean another conda-package entirely?

> This should be dependent on some environment variable.

If python is used instead of pypy3, travis fails as memory is exceeded. I think that pypy3 is the only option at the moment if we want to enable the 100T and 200T devices.
Unless we just build the data files somewhere else

Reply:
I mean rather than hard code `pypy3`, use an environment value which can be set to `pypy3` dependent on if `pypy3` is available or not (or if running on travis).

Reply:
I introduced ``USE_PYPY`` environment variable.
If it is set, then script will use ``pypy3`` if not, it will use ``python``.